test/blackbox-tests/test-cases/melange/odoc.t

```diff
@@ -0,0 +1,42 @@
+Test docs generation with Melange
```

Comment:
Could you add a dependency on the odoc package in this test?

Reply:
Done, assuming I correctly understood what you were asking for